src/DetailsView/tab-stops-requirements-with-instances.scss

```diff
@@ -0,0 +1,16 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+@import '../common/styles/colors.scss';
+@import '../common/styles/fonts.scss';
+
+.collapsible-requirement-details-group {
+    box-shadow: 0px 1px 0px rgba(0, 0, 0, 0.08);
+
+    :global(.requirement-detail) {
```

Comment:
I'm not sure this will work if this applies to the requirement-detail in this PR (since that will be generated via css-modules).